### ISSUE TITLE
jquery loading problems on latest can version

### ISSUE THREAD

**Issue by Alex0007** [canjs/canjs#2241]
```
"Potentially unhandled rejection [2] Error loading "jquery@2.2.0#jquery" at http://localhost:7000/node_modules/jquery/jquery.js
Error loading "jquery@2.2.0#jquery" from "undefined" at <Anonymous Module 1>```

Reproduce steps:
1. git clone git@github.com:whitecolor/can-hot.git && cd can-hot && npm i && npm run start-open
2. Everything works
3. Upgrade can to 2.3.14 and see jquery loading error in console

**Comment by justinbmeyer** [comment]
@Alex0007 yes, the peerDeps were removed, but not added to `devDeps` so the package wasn't working.  I've got a pull request in right now that should fix it.

**Comment by justinbmeyer** [comment]
https://github.com/canjs/canjs/pull/2242

**Comment by daffl** [comment]
Fix has been published on NPM as 2.3.15
